ansible_builder/ee_schema.py

```diff
@@ -152,10 +152,28 @@
             "type": "object",
             "additionalProperties": False,
             "properties": {
-                "prepend": {
+                "prepend_base": {
                     "anyOf": [{"type": "string"}, {"type": "array"}],
                 },
-                "append": {
+                "append_base": {
+                    "anyOf": [{"type": "string"}, {"type": "array"}],
```

Comment:
This should probably be a shared `stringOrListOfString` type that we can just reference everywhere instead of re-declaring it everywhere. It also needs to use something like `{"type": "array", "items": {"type": "string"}}` instead of untyped `array` to assert that the elements of the array are strings. Otherwise I can stuff subarrays, numbers, subdicts, whatever (which will usually blow up later in "fun" ways).

Reply:
👍🏻